eng/Versions.props

```diff
@@ -318,6 +318,7 @@
       https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json;
       https://dotnet.myget.org/F/system-commandline/api/v3/index.json;
       https://dotnetfeed.blob.core.windows.net/dotnet-coreclr/index.json;
+      https://pkgs.dev.azure.com/PublicArtifactsPreview/vside/_packaging/VSIDEPublicFeed@Local/nuget/v3/index.json
```

Comment:
Can we remove a feed at the same time? I also see "preview" in this URL -- is it going to change at some point?

Reply:
The Editor team suggested to use `pkgs.dev.azure.com` instead of `vside.myget.org`. They said it is easier for them to publish updates there regularly. This is the purpose of the PR.

Reply:
Can we remove vside.myget.org then?

Reply:
For total transparency, this feed is now in Azure DevOps beta testing, and WILL need to change at some point in the future. We're trying to use Roslyn as a proof that things are working well.

Reply:
> Can we remove vside.myget.org then?

Already :-)

Reply:
> and WILL need to change at some point in the future

What's the timeline for that? And what does this mean for servicing? We can't be adding feeds that are going to disappear since then we won't be able to build our product.

Reply:
(and I'm sympathetic to that we may need to test things, let's just make sure we don't leave this dangling and screw up servicing.)